library/breadcrumb.php

```diff
@@ -0,0 +1,40 @@
+<?php
+function the_breadcrumb() {
+    global $post;
+    echo '<ul class="breadcrumbs" role="menubar" aria-label="breadcrumbs">';
```

Comment:
Need to add this into the ul for SEO/Schema 

itemscope="itemscope" itemtype="http://schema.org/WebPageElement"